std_msgs/Header header
stargazer/MarkerPose[] marker_poses